openshift-ci/build-root/Dockerfile

```diff
@@ -1,11 +1,11 @@
 # Dockerfile to bootstrap build and test in openshift-ci
 
-FROM registry.ci.openshift.org/openshift/release:golang-1.19
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
 
 ARG KUBECTL_KUTTL_VERSION=0.12.1
 ARG OPERATOR_SDK_VERSION=1.22.2
 
-RUN yum -y install dnf httpd-tools podman docker
+RUN dnf -y install dnf httpd-tools podman docker
```

Comment:
```suggestion
RUN dnf -y install httpd-tools podman docker
```
Or
 
```suggestion
RUN yum -y install httpd-tools podman docker
```

Reply:
Thanks @deepsm007, addressed

```
/bin/sh: dnf: command not found
```

But locally this does pass, not sure what's happening

Reply:
try with `yum -y install httpd-tools podman docker `

Reply:
Still failing

Reply:
For now, I am trying this without installing httpd-tools podman and docker. If its not required by our tests, I will remove them. But if that's the case, why do we need to update the image? This issue was occurring even when the image was not changed